Makefile

```diff
@@ -20,7 +20,7 @@ example-ounit-test:
 	@dune exec example/ounit/QCheck_ounit_test.exe
 
 example-runner:
-	@dune exec example/QCheck_runner_test.exe
+	@dune exec example/QCheck_runner_test.exe -- --debug-shrink=log.tmp
```

Comment:
this is more a topic for discussion, not a review, but here `Sys.getenv` might also be quite convenient compared to passing parameters on the CLI (which needs to be negociated with each runner, including alcotest).